bin/doctest

```diff
@@ -11,7 +11,8 @@ const flags = idx >= 0 && idx < args.length - 1;
 require ('child_process')
 .spawn (
   process.execPath,
-  [...(flags ? args[idx + 1].split (/\s+/) : []),
+  ['--experimental-vm-modules',
```

Comment:
Would you also need to add `--suppress-warnings` again to not have users warned about experimental feature use?

Reply:
I don't have a strong feeling one way or the other. Do you suggest suppressing the warning?

Reply:
I guess if you don't mind the warning being there, it's probably better not to suppress _all_ warnings.